client/app/scripts/superdesk/itemList/itemList.js

```diff
@@ -383,6 +408,73 @@ function(ItemList, notify, itemPinService, gettext, $timeout) {
             });
         }
     };
+}])
+.directive('sdRelatedItemListWidget', ['ItemList', 'notify', 'gettext',
+function(ItemList, notify, gettext) {
+    return {
+        scope: {
+            options: '=',
+            itemListOptions: '=',
+            actions: '='
+        },
+        templateUrl: 'scripts/superdesk/itemList/views/relatedItem-list-widget.html',
+        link: function(scope, element, attrs) {
+            scope.items = null;
+            scope.processedItems = null;
+            scope.maxPage = 1;
+            scope.selected = null;
+
+            var oldSearch = null;
```

Comment:
I don't think this is used anywhere in this widget. Please remove copied but unused code.